subprojects/kotlin-dsl/src/test/kotlin/org/gradle/kotlin/dsl/execution/CombinatorsTest.kt

```diff
@@ -0,0 +1,275 @@
+/*
+ * Copyright 2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.kotlin.dsl.execution
+
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.Matchers.equalTo
+import org.jetbrains.kotlin.lexer.KtTokens
+import org.jetbrains.kotlin.lexer.KtTokens.IDENTIFIER
+import org.junit.Test
+
+
+class CombinatorsTest {
+
+    private
+    val combinator = Combinator(ignoresComments = false, ignoresNewline = false)
```

Comment:
It is a little ironic that the test class was renamed from `CombinatorTest` to `CombinatorsTest` and it's now testing a class named `Combinator`.

Reply:
I've heard a fitting French term recently: https://en.wiktionary.org/wiki/enculeur_de_mouches#French

But on a serious note: what is the convention/common sense that applies here? 
Yes, there is a `Combinator` class in `Combinators.kt`, but the tests are for the latter (the whole thing, not just the class). So doesn't it make sense to use `CombinatorsTest`?

Reply:
:) Bear with me; I'm not an easy person to work with... or do anything with, for that matter. Ok, I will review it once again and see what I can improve.